src/main/java/com/coherensolutions/training/automation/java/web/urnezaite/ItemPage.java

```diff
@@ -0,0 +1,67 @@
+package com.coherensolutions.training.automation.java.web.urnezaite;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class ItemPage extends BasePage {
+
+    @FindBy(xpath = "//a[@title = 'View my customer account']")
+    private WebElement myAccountButton;
+
+    @FindBy(css = "a.home")
+    private WebElement homeButton;
+
+    @FindBy(css = "p#add_to_cart>button")
+    private WebElement addToCartButton;
+
+    @FindBy(css = "a#wishlist_button")
+    private WebElement addToWishlistButton;
+
+    @FindBy(css = "h1")
+    private WebElement itemsNameElement;
+
+    @FindBy(css = "p#product_reference>span")
+    private WebElement itemsReferenceElement;
+
+    private String itemsName;
+    private String itemsReference;
```

Comment:
no need to create fields of the class in this case. You can create variables directly in methods where you use them